app/packages/plugins/src/index.ts

```diff
@@ -4,16 +4,30 @@ import * as _ from "lodash";
 import React, { FunctionComponent, useEffect, useMemo, useState } from "react";
 import ReactDOM from "react-dom";
 import * as recoil from "recoil";
+import * as foc from "@fiftyone/components";
+import * as fou from "@fiftyone/utilities";
 declare global {
   interface Window {
     __fo_plugin_registry__: PluginComponentRegistry;
     React: any;
     ReactDOM: any;
     recoil: any;
     __fos__: any;
+    __foc__: any;
+    __fou__: any;
   }
 }
 
+if (typeof window !== "undefined") {
```

Comment:
Should we move these to identical if block below and de-duplicate?

Reply:
I'll deduplicate before merging...